railties/test/application/rake/notes_test.rb

```diff
@@ -126,6 +126,18 @@ def teardown
         end
       end
 
+      test 'register additional directories' do
+        app_file "app/spec/spec_helper.rb", "# TODO: note in spec"
```

Comment:
The `app` folder is already present, I think you meant to write `"spec/spec_helper.rb`, otherwise your test will pass even without the `add_to_config` call.